scripts/generate_activity_script/src/script.ts

```diff
@@ -1,48 +1,92 @@
 import { Client as DropWithdrawalManager } from "../../../integration_tests/src/generated/contractLib/dropWithdrawalManager";
 import { Client as DropWithdrawalVoucher } from "../../../integration_tests/src/generated/contractLib/dropWithdrawalVoucher";
 
+import { MsgTokenizeShares } from "@messages";
+import { MsgSend } from "cosmjs-types/cosmos/bank/v1beta1/tx";
+import { MsgDelegate } from "cosmjs-types/cosmos/staking/v1beta1/tx";
+import { MsgTransfer } from "cosmjs-types/ibc/applications/transfer/v1/tx";
+
+import { sleep } from "../../../integration_tests/src/helpers/sleep";
+
 const CORE_CONTRACT: string = process.env.CORE_CONTRACT;
 
 const MNEMONIC: string = process.env.MNEMONIC;
 const BASE_DENOM: string = process.env.BASE_DENOM;
 const FACTORY_DENOM: string = process.env.FACTORY_DENOM;
 const NEUTRON_NODE_ADDRESS: string = process.env.NEUTRON_NODE_ADDRESS;
+const TARGET_NODE_ADDRESS: string = process.env.TARGET_NODE_ADDRESS;
+
+const TARGET_CHAIN_PREFIX: string = process.env.TARGET_CHAIN_PREFIX;
+const TARGET_DENOM: string = process.env.TARGET_DENOM;
 
 const MAX_BOND: number = Number(process.env.MAX_BOND);
 const MAX_UNBOND: number = Number(process.env.MAX_UNBOND);
-
-const BOND_PROB: number = Number(process.env.BOND_PROB);
-const UNBOND_PROB: number = Number(process.env.UNBOND_PROB);
-const WITHDRAW_PROB: number = Number(process.env.WITHDRAW_PROB);
+const MAX_LSM_PROCESS: number = Number(process.env.MAX_LSM_PROCESS);
+
+const IBC_CHANNEL_TO: string = process.env.IBC_CHANNEL_TO;
+const IBC_CHANNEL_FROM: string = process.env.IBC_CHANNEL_FROM;
+
+const BOND_PROB: number = isNaN(Number(process.env.BOND_PROB))
+  ? 0
+  : Number(process.env.BOND_PROB);
+const UNBOND_PROB: number = isNaN(Number(process.env.UNBOND_PROB))
+  ? 0
+  : Number(process.env.BOND_PROB);
+const WITHDRAW_PROB: number = isNaN(Number(process.env.WITHDRAW_PROB))
+  ? 0
+  : Number(process.env.BOND_PROB);
+const PROCESS_LSM_PROB: number = isNaN(Number(process.env.PROCESS_LSM_PROB))
+  ? 0
+  : Number(process.env.PROCESS_LSM_PROB);
 
 /*
  * Each of given probabilities should be in interval [0, 1]
  * According to given probabilities relevant actions will be choosed with appropriate probability
  */
 if ((0 <= BOND_PROB && BOND_PROB <= 1) === false) {
-  console.error(`0 <= BOND_PROB(${BOND_PROB}) <= 1 != true`);
-  process.exit(1);
+  throw `BOND_PROB should be in interval 0 <= BOND_PROB(${BOND_PROB}) <= 1`;
 }
 if ((0 <= UNBOND_PROB && UNBOND_PROB <= 1) === false) {
-  console.error(`0 <= UNBOND_PROB(${UNBOND_PROB}) <= 1 != true`);
-  process.exit(1);
+  throw `UNBOND_PROB should be in interval 0 <= UNBOND_PROB(${UNBOND_PROB}) <= 1`;
 }
 if ((0 <= WITHDRAW_PROB && WITHDRAW_PROB <= 1) === false) {
-  console.error(`0 <= WITHDRAW_PROB(${WITHDRAW_PROB}) <= 1 != true`);
-  process.exit(1);
+  throw `WITHDRAW_PROB should be in interval 0 <= WITHDRAW_PROB(${WITHDRAW_PROB}) <= 1`;
+}
+if ((0 <= PROCESS_LSM_PROB && PROCESS_LSM_PROB <= 1) === false) {
```

Comment:
```suggestion
if (PROCESS_LSM_PROB < 0 || PROCESS_LSM_PROB > 1) {
```
also 
(...something...) === false 
the same as 
!(...something...)